core/environment/environment.go

```diff
@@ -186,9 +186,22 @@ func newEnvironment(userVars map[string]string, newId uid.ID) (env *Environment,
 					env.workflow.GetVars().Set("run_number", rnString)
 					env.workflow.GetVars().Set("runNumber", rnString)
 
-					runStartTime := strconv.FormatInt(time.Now().UnixMilli(), 10)
-					env.workflow.SetRuntimeVar("run_start_time_ms", runStartTime)
-					env.workflow.SetRuntimeVar("run_end_time_ms", "") // we delete previous EOR
+					runStartTime := time.Now()
+					runStartTimeS := strconv.FormatInt(runStartTime.UnixMilli(), 10)
```

Comment:
```suggestion
					runStartTimeStr := strconv.FormatInt(runStartTime.UnixMilli(), 10)
```
so it is not ambiguous with "S" for "Seconds"?

Same suggestion for similar occurrences elsewhere.